tools/isobuild/bundler.js

```diff
@@ -1522,11 +1522,17 @@ class ClientTarget extends Target {
     if (this.arch === 'web.cordova') {
       const { WebAppHashing } =
         loadIsopacket('cordova-support')['webapp-hashing'];
+
       const cordovaCompatibilityVersions =
         _.object(_.map(CORDOVA_PLATFORM_VERSIONS, (version, platform) => {
-          const hash = WebAppHashing.calculateCordovaCompatibilityHash(
-            version,
-            this.cordovaDependencies);
+          let cordovaDependencies = Object.assign({}, this.cordovaDependencies);
+          let pluginsExcludedFromCompatibilityHash = process.env.CORDOVA_COMPATIBILITY_VERSION_EXCLUDE || '';
+          cordovaDependencies = _.omit(cordovaDependencies, pluginsExcludedFromCompatibilityHash.split(';'));
```

Comment:
Instead of re-assigning `cordovaDependencies`, is there a reason we can't switch the order the variables are created and then switch them to `const`s?

```js
const pluginsExcludedFromCompatibilityHash =
  (process.env.CORDOVA_COMPATIBILITY_VERSION_EXCLUDE || '')
  .split(';');
const cordovaDependencies =
  Object.assign(Object.create(null),
    _.omit(this.cordovaDependencies, pluginsExcludedFromCompatibilityHash));
```